exporters/otlp/CMakeLists.txt

```diff
@@ -27,6 +27,18 @@ if(WITH_OTLP_GRPC)
                                             protobuf::libprotobuf gRPC::grpc++)
 
   list(APPEND OPENTELEMETRY_OTLP_TARGETS opentelemetry_exporter_otlp_grpc)
+
+  add_library(opentelemetry_exporter_otlp_grpc_log
+              src/otlp_grpc_log_exporter.cc)
+
+  set_target_properties(opentelemetry_exporter_otlp_grpc_log
+                        PROPERTIES EXPORT_NAME otlp_grpc_log_exporter)
+
+  target_link_libraries(
+    opentelemetry_exporter_otlp_grpc_log
+    PUBLIC opentelemetry_otlp_recordable protobuf::libprotobuf gRPC::grpc++)
```

Comment:
`protobuf::libprotobuf` is already PUBLIC linked by `opentelemetry_proto` , and `opentelemetry_proto` is PUBLIC linked by `opentelemetry_otlp_recordable`. Maybe PUBLIC linking `protobuf::libprotobuf` into `opentelemetry_exporter_otlp_grpc_log` and `opentelemetry_exporter_otlp_grpc` is redundant?

Reply:
Good point. I removed `protobuf::libprotobuf` for both `opentelemetry_exporter_otlp_grpc` and `opentelemetry_exporter_otlp_grpc_log` targets.